src/main/java/org/jenkinsci/plugins/DependencyTrack/DependencyTrackPublisher.java

```diff
@@ -15,6 +15,7 @@
  */
 package org.jenkinsci.plugins.DependencyTrack;
 
+import hudson.AbortException;
```

Comment:
Per the documentation in BuildStep.perform(), an AbortException should be thrown if an error has been handled and diagnosed by the build step.  Using AbortException (apparently) suppresses the printing of a stack trace to the build output. Other exceptions that cause build aborts cause a stack trace to be a dumped.